src/app/payment/paymentMethod/DigitalRiverPaymentMethod.spec.tsx

```diff
@@ -20,16 +20,18 @@ describe('when using Digital River payment', () => {
     let defaultProps: PaymentMethodProps;
     let localeContext: LocaleContextType;
     let PaymentMethodTest: FunctionComponent<PaymentMethodProps>;
+    const DigitalRiverMethod = getPaymentMethod();
+    DigitalRiverMethod.config.isVaultingEnabled = true;
 
     beforeEach(() => {
         defaultProps = {
-            method: getPaymentMethod(),
+            method: DigitalRiverMethod,
```

Comment:
This shouldn't be capitalised, only classes and types are.

```suggestion
            method: digitalRiverMethod,
```